src/main/java/by/onliner/util/ActionsHelper.java

```diff
@@ -7,15 +7,10 @@
 /***
  * Wrapper class for actions moves by mouse to the defined element
  */
-public class MoveTo {
-
-    protected static Actions actions = new Actions(DriverManager.getDriver());
+public class ActionsHelper {
 
     public static void moveToElementAndClick(WebElement webElement) {
+        Actions actions = new Actions(DriverManager.getDriver());
```

Comment:
why have you added it here?

for now, you have only one method in this class but i am fairly sure more methods will be added in the future

Reply:
⭐

Reply:
?

Reply:
please check my previous comment ^^